backend/blueprint/api/controllers/homePage.py

```diff
@@ -1,10 +1,10 @@
 from flask import Blueprint, jsonify
-from api.utils.response import Response
+from utils.response import Response
 
 router = Blueprint('homePage', __name__)
 
 
-@router.route('/', methods=['POST'])
+@router.route('/', methods=['GET', 'POST'])
```

Comment:
Not needed

Reply:
So if We request / then User should able to see something. IMO, Documentation should be served at this page and also an indication that API is working fine